### PR TITLE
fix(curriculum): remove `-webkit-clip-path` from sr-only and its test

### LINKED ISSUE #49792: Remove -webkit-clip-path property from sr-only class
### Describe the Issue

The Building a Balance Sheet course uses the `sr-only` class (beginning on [Step 31](https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-31)) to visually hide text for accessibility. This class is using the fallback `-webkit-clip-path` property which is not needed any more as all browsers have supported `clip-path` for quite a while now. Furthermore, all modern recommended patterns for visually hiding text do not include it.

[The anatomy of visually-hidden](https://www.tpgi.com/the-anatomy-of-visually-hidden/)
[WebAIM: Invisible Content Just for Screen Reader Users](https://webaim.org/techniques/css/invisiblecontent/)
[CSS-Tricks: Inclusively Hidden](https://css-tricks.com/inclusively-hidden/)

I think it would be a good idea if we removed it from the course as campers are never going to use `-webkit-clip-path` and do not need to learn it here. It will also allow us to remove a regex test for step 31 (see PR #49546). 

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-31

### Your code

N/A

### Expected behavior

Course does not use `-webkit-clip-path` in `sr-only` class.

### Screenshots

_No response_

### System

N/A

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Jiajun-Jiao**
- Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #49792 

Made this change because -webkit-clip-path is no longer needed due to improved browser support for the standard clip-path property.

This is the screenshot after change:
Screenshot 2023-03-22 at 10 30 20

All the following steps are also modified.
The files modified are all under 14-responsive-web-design-22 => learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet package.

These bullet points are achieved:
- Remove mention/instruction of -webkit-clip-path.
- Remove test looking for the property
- Remove the property from all following lessons/steps
- Only change the English files

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/6201a59be346d399c21d10b1.md
Comment (ShaunSHamilton) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/6201a59be346d399c21d10b1.md:1]: As this is the last file for a project, it contains both the boilerplate (seed) and the solution code.

Please remove the `-webkit-clip-path` property here: https://github.com/freeCodeCamp/freeCodeCamp/pull/49796/files#diff-510dc1b574de1b5414ff2a90ca3fccbaeade55467d0f4478af833c3e09ff867bR441

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Thank you for taking the time to contribute.

**Requested Changes**

1) Step `43` is missing (it should also be changed)
2) The final step needs the solution code also updated

**Review by ShaunSHamilton — APPROVED** [review]
Thank you for taking care of those changes. This looks good to me 👍

**Review by gikf — APPROVED** [review]

**Comment by gikf** [comment]
Hi @Jiajun-Jiao. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**